scripts/experiments_images/SkipGANomaly_run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#SBATCH --time=24:00:00
+#SBATCH --nodes=1 --ntasks-per-node=2 --cpus-per-task=1
+#SBATCH --gres=gpu:1
+#SBATCH --mem=20G
+
+MAX_SEED=$1
+DATASET=$2
+ANOMALY_CLASSES=$3
+
+module load Julia/1.4.1-linux-x86_64
+module load Python/3.8.2-GCCcore-9.3.0
```

Comment:
Same points as in the case of the ganomaly script

Reply:
fixed